compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticCode.java

```diff
@@ -318,6 +318,7 @@
     UNSUPPORTED_METHOD_INVOCATION_XML_NAV("method.invocation.in.xml.navigation.expressions.not.supported"),
     DEPRECATED_XML_ATTRIBUTE_ACCESS("deprecated.xml.attribute.access.expression"),
     UNSUPPORTED_INDEX_IN_XML_NAVIGATION("indexing.within.xml.navigation.expression.not.supported"),
+    DEPRECATED_XML_CHILD_ACCESS("deprecated.xml.child.access"),
```

Comment:
Can we remove this since it is no longer used?

Reply:
Done